### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by david-puglielli**
Updated install instructions. Some of the stuff in Future Plans and FAQs also needs updating.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -278,10 +289,10 @@ The following instructions assume a clean environment and show how to install PH
 
 	sudo apachectl restart 
 
-*Note to RedHat users: SELinux is installed by default and runs in Enforcing mode. To allow Apache to connect to database through SELinux, do this `sudo setsebool -P httpd_can_network_connect_db 1`     
+*Note to RedHat users: SELinux is installed by default and runs in Enforcing mode. To allow Apache to connect to database through SELinux, run `sudo setsebool -P httpd_can_network_connect_db 1`     
 
 ### Step 6: Create your sample app
-Navigate to `/var/www/html` (`/usr/local/var/www/htdocs` on Mac) and create a new file called testsql.php. Copy and paste the following code in tetsql.php and change the servername, username, password and databasename.
+Navigate to `/var/www/html` (`/usr/local/var/www/htdocs` on Mac) and create a new file called testsql.php. Copy and paste the following code into tetsql.php and change the servername, username, password and databasename.
```
Comment (yitam) [inline README.md]: suggestion: use `testsql.php` instead of `tetsql.php`?

File: README.md [hunk 2]
```diff
@@ -395,15 +406,15 @@ Thank you!
 ## FAQs
 **Q:** Can we get dates for any of the Future Plans listed above?
 
-**A:** At this time, Microsoft is not able to announce dates. We are working extremely hard to release future versions of the driver. We will share future plans once they solidify over the next few weeks. 
+**A:** At this time, Microsoft is not able to announce dates. We are working extremely hard to release future versions of the driver. We will share future plans as appropriate. 
 
 **Q:** What's next?
 
-**A:** On July 20, 2016 we released the early technical preview for our PHP Driver. We will continue releasing frequent technical previews until we reach production quality.
+**A:** On July 6, 2017 we released the production release version 4.3.0 of our PHP Driver. We will continue working on our future plans and releasing frequent updates.
```
Comment (yitam) [inline README.md]: should we specify that frequent updates are previews?

Comment (Hadis-Fard) [inline README.md]: ggo d catch, no we don't need that

Comment (david-puglielli) [inline README.md]: We don't need to mention frequent updates you mean?

File: README.md [hunk 3]
```diff
@@ -1,456 +1,492 @@
...
+**Debian 8**
+
+    sudo su
+    apt-get install curl apt-transport-https
+    curl https://www.dotdeb.org/dotdeb.gpg | apt-key add -
+    echo "deb http://packages.dotdeb.org jessie all" >> /etc/apt/sources.list
+    echo "deb-src http://packages.dotdeb.org jessie all" >> /etc/apt/sources.list
+    apt-get update
+    apt-get install -y php7.0 php-pear php7.0-dev php7.0-xml
+
```
Comment (yitam) [inline README.md]: Inconsistent spacing for Deb 8 when compared to others...

File: README.md [hunk 4]
```diff
@@ -1,456 +1,492 @@
...
+## Install (UNIX)
+The following instructions assume a clean environment and show how to install PHP 7.x, Microsoft ODBC driver, apache, and Microsoft PHP drivers on Ubuntu 15, 16, RedHat 7, Debian 8, and Mac OS X. 
+
+### Step 1: Install PHP7+ 
+
+#### PHP 7.0
+
+**Ubuntu 15.04, Ubuntu 15.10**
+
```
Comment (yitam) [inline README.md]: Do we still support Ub 15.04? Also, how come there is no instructions for Ub 15 for PHP 7.1?

Comment (yitam) [inline README.md]: I think our README is getting way too long, which somewhat defeats the purpose. Isn't it time to move all these instructions to [End to End instructions](https://github.com/Microsoft/msphpsql/wiki/Install-and-configuration#EndToEnd)?

Comment (david-puglielli) [inline README.md]: I've removed references to Ubuntu 15.04. PHP 7.1 is not available for Ubuntu 15 in package form (or it wasn't last time I checked). I'll add a note.

File: README.md [hunk 5]
```diff
@@ -1,456 +1,494 @@
...
-
-## Announcements
-
- Please visit the [blog][blog] for more announcements.
-
-
-## Build (Windows)
-
-Note: if you prefer, you can use the pre-compiled binary found [HERE](https://github.com/Azure/msphpsql/releases)
-
```
Comment (yitam) [inline README.md]: Please update the GitHub url here

File: README.md [hunk 6]
```diff
@@ -1,456 +1,494 @@
...
-- A Web server such as Internet Information Services (IIS) is required. Your Web server must be configured to run PHP
-- [Microsoft ODBC Driver 11][odbc11] or [Microsoft ODBC Driver 13][odbc13]
-
-#### Enable the drivers
-
-1. Make sure that the driver is in your PHP extension directory (you can simply copy it there if you did not use nmake install).
-
-2. Enable it within your PHP installation's php.ini: `extension=php_sqlsrv.dll` and/or `extension=php_pdo_sqlsrv.dll`.  If necessary, specify the extension directory using extension_dir, for example: `extension_dir = "C:\PHP\ext"`
-
```
Comment (yitam) [inline README.md]: the dll names are different if the user download them from GitHub releases right?

**Pull request merged**